src/test/java/apicurito/tests/configuration/Component.java

```diff
@@ -5,8 +5,9 @@
 @Getter
 public enum Component {
 
-    GENERATOR("fuse-apicurito-generator"),
-    UI("apicurito-ui");
+    UI("apicurito-ui"),
```

Comment:
The generator is not deployed in the test suite?

Reply:
template installation - yes 
operator installation - no 
but anyway we do not test generator and it is not needed to test apicurito

Reply:
I see, thanks